### PR TITLE
Importing a kubernetes application into a che workspace

### PR DESCRIPTION
**Pull request by MichalMaler**
https://issues.jboss.org/browse/RHDEVDOCS-1152

adding content about: "Importing a kubernetes application into a Che workspace"

rewrote this document into .adoc
https://docs.google.com/document/d/1VZJSRh8kFpr5U8R_DDycv-4MgFqzdyZybmCcco_ShUg/edit#

### CODE REVIEW

**Comment by MichalMaler** [comment]
It's ready for SME review @slemeur @l0rd

@rkratky  @supriya-bharadwaj Can I ask you for a Peer review, please?

File: src/main/pages/che-7/hands-on-guidance/assembly_importing-a-kubernetes-application-into-a-che-workspace.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+:parent-context-of-importing-a-kubernetes-application-into-a-che-workspace: {context}
+
+[id='importing-a-kubernetes-application-into-a-che-workspace_{context}']
+= Importing a Kubernetes application into a Che workspace
+
+:context: importing-a-kubernetes-application-into-a-che-workspace
+
+This section describes how to add a Kubernetes application in a Che workspace.
```
Comment (slemeur) [inline src/main/pages/che-7/hands-on-guidance/assembly_importing-a-kubernetes-application-into-a-che-workspace.adoc]: "how to import"

File: src/main/pages/che-7/hands-on-guidance/assembly_importing-a-kubernetes-application-into-a-che-workspace.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+For demonstration purpose, the section uses a sample Kubernetes application having the following two pods:
+
+* A NodeJS application specified by this link:https://raw.githubusercontent.com/redhat-developer/devfile/master/samples/web-nodejs-with-db-sample/nodejs-app.yaml[yaml]
+* A mongo DB pod specified by this link:https://raw.githubusercontent.com/redhat-developer/devfile/master/samples/web-nodejs-with-db-sample/mongo-db.yaml[yaml]
+
+To run the application  on a Kubernetes cluster:
```
Comment (slemeur) [inline src/main/pages/che-7/hands-on-guidance/assembly_importing-a-kubernetes-application-into-a-che-workspace.adoc]: space character

File: src/main/pages/che-7/hands-on-guidance/proc_adding-a-kubernetes-application-to-an-existing-workspace-using-the-dashboard.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
+
+// importing-a-kubernetes-application-into-a-che-workspace
+
+[id="adding-a-kubernetes-application-to-an-existing-workspace-using-the-dashboard_{context}"]
+= Adding a Kubernetes application to an existing workspace using the Dashboard
+
+This procedure demonstrates how to modify an existing workspace and add the Kubernetes application using the newly created devfile.
```
Comment (slemeur) [inline src/main/pages/che-7/hands-on-guidance/proc_adding-a-kubernetes-application-to-an-existing-workspace-using-the-dashboard.adoc]: "add" --> "import"

File: src/main/pages/che-7/hands-on-guidance/proc_including-a-kubernetes-application-in-a-workspace-devfile-definition.adoc [hunk 4]
```diff
@@ -0,0 +1,57 @@
+
+// importing-a-kubernetes-application-into-a-che-workspace
+
+[id="including-a-kubernetes-application-in-a-workspace-devfile-definition_{context}"]
+= Including a Kubernetes application in a workspace devfile definition
+
+// TO DO Supply URL for that file location
+The devfile format is used to define a Che workspace and its format is described in the Making a workspace portable using a devfile section. The following is an example of the simplest devfile:
```
Comment (slemeur) [inline src/main/pages/che-7/hands-on-guidance/proc_including-a-kubernetes-application-in-a-workspace-devfile-definition.adoc]: can we include a link to that section?

**Review by rkratky — APPROVED** [review]
@slemeur I added a commit with a bunch of fixes. It also addresses your comments.

**Comment by slemeur** [comment]
Thanks

**Pull request merged**